docker/Dockerfile.ubuntu.xenial

```diff
@@ -13,26 +13,23 @@ RUN apt-get update && apt-get -y install --no-install-suggests --no-install-reco
   autoconf \
   asn1c \
   automake \
+  awscli \
   bison \
   ccache \
-  cmake \
+  clang-format-6.0 \
```

Comment:
Do we need clang-format here?

Reply:
maybe this is for the case when we decide to check the formatting, not sure...